repos/system_upgrade/el7toel8/actors/detectgrubconfigerror/actor.py

```diff
@@ -0,0 +1,29 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.scanner import detect_config_error
+from leapp.models import CheckResult, GrubConfigError
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class DetectGrubConfigError(Actor):
+    """
+    Checks grub configuration for syntax error in GRUB_CMDLINE_LINUX value.
```

Comment:
```suggestion
    Check grub configuration for syntax error in GRUB_CMDLINE_LINUX value.
```
Use command/imperative style, see [PEP257](https://www.python.org/dev/peps/pep-0257/):

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

The same in the other docstrings.